Gemfile

```diff
@@ -27,6 +27,7 @@ gem "puppet", :path => File.dirname(__FILE__), :require => false
 gem "facter", *location_for(ENV['FACTER_LOCATION'] || ['> 2.0', '< 4'])
 gem "hiera", *location_for(ENV['HIERA_LOCATION'] || ['>= 2.0', '< 4'])
 gem "rake", "10.1.1", :require => false
+gem 'hocon', '~> 1.1.1'
```

Comment:
@kylog this is only used for development, right?  So it's OK to have this here and we will just, additionally, need to sort out the packaging issues we've been discussing?